app/elements/studentList.js

```diff
@@ -19,20 +19,11 @@ let StudentList = React.createClass({
 	handleFile(ev) {
 		var reader = new FileReader()
 		var file = ev.target.files[0]
+		reader.readAsText(file)
```

Comment:
You'll want to define the ` onload ` handler *before* you call the thing that will provide events to the event handlers.